tools/run_baas.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+PROJECT_DIR=$(git rev-parse --show-toplevel)
+MDBREALM_TEST_SERVER_TAG=$(grep MDBREALM_TEST_SERVER_TAG ${PROJECT_DIR}/dependencies.list |cut -f 2 -d=)
+docker run --rm -p 9090:9090 -v ~/.aws/credentials:/root/.aws/credentials -it docker.pkg.github.com/realm/ci/mongodb-realm-test-server:${MDBREALM_TEST_SERVER_TAG}
```

Comment:
If you want, you can also just run `./evergreen/install_baas.sh -w baas-work-dir` with your AWS credentials in the correct environment variables and it will the latest version of baas deployed to prod.

Reply:
ok, but I just wanted a script that starts the version we refer to in the dependencies. I think that is the one that is run in Jenkins CI.

Reply:
If I run `./evergreen/install_baas.sh -w baas-work-dir`, I get `Unsupported platform linuxmint 19.3`

Reply:
I will rename the script.